docs/specification.rst

```diff
@@ -1300,7 +1301,7 @@ together using the ``$glue`` argument as a separator between each.
     - ``join(`, `, @)``
     - "a, b"
   * - ``["a", "b"]``
-    - ``join(````, @)``
+    - :literal:`join(\`\`, @)`
```

Comment:
Similarly, using the `:literal:` role here so that we can escape the back-quotes.